rundeckapp/grails-spa/packages/ui-trellis/src/app/components/activity/tests/activityList.spec.ts

```diff
@@ -0,0 +1,293 @@
+import {
+  rundeckServiceMock,
+  rundeckClientMock,
+  axiosMock,
+  i18nMocks,
+  mockEventBus,
+} from "./mock";
+import { mount, VueWrapper } from "@vue/test-utils";
+
+import ActivityList from "../activityList.vue";
+import ActivityFilter from "../activityFilter.vue";
+import OffsetPagination from "../../../../library/components/utils/OffsetPagination.vue";
+
+import { setupRundeckContext } from "./setupRundeckContext";
+import { GlobalOptions } from "./type";
+
+jest.mock("../../../../library/rundeckService", () => rundeckServiceMock);
+jest.mock("@rundeck/client", () => rundeckClientMock);
+jest.mock("axios", () => axiosMock);
+type ActivityListInstance = InstanceType<typeof ActivityList>;
+const defaultProps = {
+  pagination: {
+    total: 10,
+    offset: 0,
+    max: 10,
+  },
+  reports: [],
+  query: {},
+  eventBus: mockEventBus,
+  running: {
+    executions: [],
+  },
+  bulkEditMode: true,
+  bulkSelectedIds: [],
+};
+const mountActivityList = async (
+  props = {},
+  globalOptions: GlobalOptions = {}
+) => {
+  const wrapper = mount(ActivityList, {
+    props: {
+      ...defaultProps,
+      ...props,
+    },
+    global: {
+      components: {
+        OffsetPagination,
+        ActivityFilter,
+      },
+      stubs: {
+        modal: true,
+        "i18n-t": true,
+        ActivityFilter: true,
+        Modal: { template: "<div><slot></slot></div>" },
+        ProgressBar: { template: "<div><slot></slot></div>" },
+        btn: {
+          template: `<button :disabled="disabled" data-test-id="activity-list-delete-selected-executions">Delete Selected Executions</button>`,
+          props: ["disabled"],
+        },
+
+        i18n: true,
+        tooltip: true,
+        ...globalOptions.stubs,
+      },
+      directives: {
+        tooltip: () => {},
+      },
+      mocks: {
+        ...i18nMocks,
+        ...globalOptions.mocks,
+      },
+      ...globalOptions,
+    },
+  });
+  await wrapper.vm.$nextTick();
+  return wrapper as VueWrapper<ActivityListInstance>;
+};
+beforeAll(() => {
+  jest.useFakeTimers();
+  setupRundeckContext();
+});
+afterAll(() => {
+  jest.useRealTimers();
+});
+afterEach(() => {
+  jest.clearAllMocks();
+});
+describe("ActivityList Component", () => {
+  it("renders the component correctly", async () => {
+    const wrapper = await mountActivityList();
+    expect(wrapper.exists()).toBe(true);
+  });
+  it("renders loading area", async () => {
+    const wrapper = await mountActivityList({ loading: true });
+    expect(wrapper.find('[data-test-id="loading-area"]').exists()).toBe(true);
+  });
+  it("displays an error message when load fails", async () => {
+    const wrapper = await mountActivityList();
+    const loadError = "An Error Occurred";
+    await wrapper.setProps({ loadError });
```

Comment:
ActivityList doesn't have a prop for loadError - if this test is passing, it's a false positive. Need to check how the error message is being set in the component and rewrite this test.